p/themes/Pafat/pafat.css

```
@@ -1168,6 +1168,7 @@ a.btn {
 
 	.nav_menu .stick {
 		margin: 0 10px;
+		min-width: max-content;
```

```suggestion
		min-width: 0;
```

Should achieve the same, while being lighter and more back-compatible